deploy/crds/infinispan.org_infinispans_crd.yaml

```diff
@@ -476,6 +476,15 @@ spec:
                       type: string
                     certServiceName:
                       type: string
+                    clientCert:
+                      description: ClientCertType specifies all the support client certificate validations
```

Comment:
```suggestion
                      description: Specifies a client certificate validation mechanism.
```

Reply:
Updated to "ClientCertType specifies a client certificate validation mechanism." as this follows the convention established elsewhere.

FYI @oraNod this CRD content and that found in `operator-install.yaml` is generated from the values in `pkg/apis/infinispan/v1/infinispan_types.go`, so you can just update the value there and execute `make codegen` to update these files.

Reply:
thanks for the tip, @ryanemerson I've realized the need to improve edits to field descriptions. I'm going to change up my review workflow so I'm not adding headaches. cheers.